frontend/src/general/dialog/__tests__/Dialog.test.jsx

```diff
@@ -29,6 +38,43 @@ describe("<Dialog/> Class Component", () => {
       expect(wrapper.find(DialogBody)).toHaveLength(1);
       expect(wrapper.find(DialogButtons)).toHaveLength(1);
     });
+
+    describe("if an actionButton is passed as a prop", () => {
+      it("button2 prop should be passed with the actionButton", () => {
+        const dialogButtonsComponent = wrapper.find(DialogButtons);
+        expect(dialogButtonsComponent.props().button2).toBe(actionButton);
+      });
+    });
+
+    describe("if no actionButton is passed as a prop", () => {
+      const wrapperNoActionButton = Enzyme.shallow(
+        <Dialog title={dummyTitle}>{testBody}</Dialog>
+      );
+
+      it("button2 prop should be undefined", () => {
+        const dialogButtonsComponent = wrapperNoActionButton.find(
+          DialogButtons
+        );
+
+        expect(dialogButtonsComponent.props().button2).toBeUndefined();
+
+        wrapperNoActionButton.unmount();
```

Comment:
Oh wait yeah I should have defined it in the it() statement, I think that was just overlooked by me since both it() functions actually test different wrappers.